openpype/hosts/maya/tools/mayalookassigner/app.py

```diff
@@ -263,14 +263,13 @@ def on_process_selected(self):
                 for standin in arnold_standins:
                     if standin in nodes:
                         arnold_standin.assign_look(standin, subset_name)
+                nodes = list(set(item["nodes"]).difference(arnold_standins))
```

Comment:
Ok, so this logic actually is wrong if both mtoa and vrayformaya are loaded. The discarded v-ray proxies at that point in time are not discarded anymore. As such, I believe this (and the line for vray proxies) should be:

```suggestion
                nodes = list(set(nodes).difference(arnold_standins))
```

Right?

Reply:
Yeah, sure we can do that.